models/model_predict.ipynb

```diff
@@ -0,0 +1,580 @@
+   "source": [
+    "from clarifai.client.app import App\n",
+    "\n",
+    "all_llm_community_models = App().list_models(filter_by={\"query\": \"LLM\",\n",
+    "                                                        \"model_type_id\": \"text-to-text\"}, only_in_app=False)\n",
+    "\n",
+    "for llm_model in all_llm_community_models:\n",
+    "  print(\"Model ID \", llm_model.id)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Get Model Details\n",
+    "\n",
+    "Below cell is how to know more on details of model(description, usecases..etc) and info on training or other inference parameters(eg: temperature, top_k, max_tokens..etc for LLMs)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from clarifai.client.user import User\n",
+    "gpt_4_model = User(user_id=\"openai\").app(app_id=\"chat-completion\").model(model_id=\"GPT-4\")\n",
+    "print(gpt_4_model)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Model Inference"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Model class objects can be inititalised by providing its URL or also by defining respective `user_id`, `app_id` and `model_id`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from clarifai.client.model import Model\n",
+    "model = Model(\"https://clarifai.com/clarifai/main/models/general-image-recognition\")\n",
+    "                    # or\n",
+    "model = Model(user_id=\"clarifai\", app_id=\"main\", model_id=\"general-image-recognition\")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Text"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Text-to-Text\n",
+    "\n",
+    "Model takes text data as input by filename or bytes or url.\n",
+    "\n",
+    "Below is an example of how you would send text bytes/filepath and receive predictions from Clarifai's hosted models"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Predict by Bytes"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Text Generation** - Generating text from an LLM"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Asking Claude2 to write a tweet on future of AI\n",
+    "model_url = \"https://clarifai.com/anthropic/completion/models/claude-v2\"\n",
+    "model_prediction = Model(model_url).predict_by_bytes(b\"Write a tweet on future of AI\", input_type=\"text\")\n",
+    "print(model_prediction)\n"
```

Comment:
For maintaining convention, we can have the completion alone here, like below `print(model_prediction.outputs[0].data.text.raw)`